packages/@apphosting/adapter-angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apphosting/adapter-angular",
-  "version": "17.2.5",
+  "version": "17.2.6",
```

Comment:
why patch bump? should this be a minor version bump?

Reply:
Our adapter versioning is unconventional. The adapter major and minor versions match the respective framework versions that the adapter is compatible with, regardless of what changes we make.

https://docs.google.com/document/d/1t0GLfbX4Suhp-NY0syVeI2rjlizhyTKQJU9fkWe4bFs/edit?resourcekey=0-DC0j6pH5qpCObySMfcKV0A&tab=t.0#heading=h.49zqbtr2osgh